tools/cbpManager.deploy/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+services:
+  cbpmanager-nginx:
+    build: ./nginx-forwarder
+    image: cbpmanager-nginx
+    container_name: cbpmanager_nginx_container
+    networks:
+     - cbpmanager_net
+  cbpmanager:
+    build: ./cbpmanager
+    image: cbpmanager
+    container_name: cbpmanager_container
+    ports:
+      - 3838:3838
```

Comment:
As you are using shinyproxy there is no actual need to expose this port on the host system.